flashmodels/accelerators/acc_baichuan_accelerator.py

```diff
@@ -17,7 +16,7 @@ def accelerate_internal(self, model, loader):
                 raise NotImplementedError("resume_from_checkpoint.")
 
             config = self.get_config(model)
```

Comment:
can we support spmd_fsdp for other models?

Reply:
Not test yet, will add after test